src/ProjectNeon/Assets/Features/Effects/AllEffects.cs

```diff
@@ -27,6 +27,7 @@ public static class AllEffects
         { EffectType.ShieldAttackedOnAttack, e => new EffectOnAttacked(new SimpleEffect((src, m) => m.GainShield(e.IntAmount * src.State.Toughness()))) },
         { EffectType.DamageAttackerOnAttack, e => new EffectOnAttacker(new SimpleEffect((src, m) => m.TakePhysicalDamage(e.IntAmount * src.State.Attack()))) },
         { EffectType.StealLifeNextAttack, e => new Recurrent(new StealLife(e.IntAmount), 1)},
+        { EffectType.InterceptAttackForTurns, e => new Recurrent(new InterceptAttack(), e.NumberOfTurns)},
```

Comment:
Supposed to be for Number of Turns, not number of instances.

Reply:
@SilasReinagel Sorry, my bad. Fixed